config/config.toml.example

```diff
@@ -361,7 +361,7 @@ max-retry-count = 256
 
 [stmt-summary]
 # enable statement summary.
-enable = false
+enable = true
```

Comment:
Could we add a test to config_test.go?

Reply:
Refer to https://github.com/pingcap/tidb/blob/master/config/config_test.go#L226, is it enough？

Reply:
We can change this config value in this test.